pkg/routing/interfaces.go

```diff
@@ -30,12 +30,8 @@ type MessageSource interface {
 type ParticipantInit struct {
 	Identity      livekit.ParticipantIdentity
 	Name          livekit.ParticipantName
-	Metadata      string
```

Comment:
`Grants` now supersedes all of these attributes.

Reply:
Nice!!!